MarkupGatewayPlugin.inc.php

```diff
@@ -257,25 +257,98 @@ function fetch($args, $request) {
 	 */
 	protected function _buildMetadata($journal, $submission)
 	{
-		$locale = AppLocale::getLocale();
-
+		$locale = ($submission->getLanguage() != '') ? $submission->getLanguage() : $journal->getPrimaryLocale();
+		
+		$publishedArticleDao = DAORegistry::getDAO('PublishedArticleDAO');
+		$issueDao = DAORegistry::getDAO('IssueDAO');		
+		$sectionDao = DAORegistry::getDAO('SectionDAO');
+		$userGroupDao = DAORegistry::getDAO('UserGroupDAO');
+		
+		/* Authors */
 		$authors = array_map(function($author)
 		{
 			return array (
-					'name' => $author->getFullName(),
+					'firstName' => $author->getFirstName(),
+					'lastName' => $author->getLastName(),
 					'email' => $author->getEmail(),
+					'orcid' => $author->getOrcid(),
+					'affiliation' => $author->getLocalizedAffiliation(),
+					'country' => $author->getCountry(),
+					'bio' => $author->getLocalizedBiography(),
+					'contribType' => $userGroupDao->getById($author->getUserGroupId()),
```

Comment:
hey @ajnyga, any update on this? I'd like to get this merged since your rewrite of the metadata parsing on the OTS side is pretty nice, but I'll hold off if you're still planning on working on it yourself.